src/handlers/comment/handlers/multiplier.ts

```diff
@@ -18,13 +18,37 @@ export const multiplier = async (body: string) => {
     return;
   }
 
-  const regex = /\/multiplier @(\w+) (\d+(?:\.\d+)?)/; // /multiplier @0xcodercrane 0.5
+  const regex = /(".*?"|[^"\s]+)(?=\s*|\s*$)/g;
+  /** You can use this command to set a multiplier for a user.
```

Comment:
Shouldn't this regular expression begin with `/multiplier`?

Reply:
This regex looks like a more global one. we also have a command type check [here](https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/comment/handlers/index.ts#L34) so i guess its fine to keep